src/components/StaticThought.tsx

```diff
@@ -210,7 +210,6 @@ const StaticThought = ({
                 ...(isTableCol1 && { maxWidth: '100%' }),
                 ...(isAttribute(value) && { fontFamily: 'monospace' }),
                 ...(ellipsizedUrl && {
-                  display: 'inline-block',
```

Comment:
I've tested this and can't find any regressions, so I will tentatively say it's safe to remove.

`inline-block` was added in the original url ellipsis implementation in  https://github.com/cybersemics/em/pull/2082/files#diff-60f5dcfc15327d5dd812d9df394c217efbedb4aa33dca782ed69d39dce811972R1818. It's not clear why, since it appears to work fine with the default `display: block`.